languages/ruby/exercises/concept/floating-point-numbers/.meta/example.rb

```diff
@@ -0,0 +1,39 @@
+module SavingsAccount
+
+  NEGATIVE_RATE = -3.213
+  SMALL_POSITIVE_RATE = 0.5
+  MEDIUM_POSITIVE_RATE = 1.621
+  LARGE_POSITIVE_RATE = 2.475
```

Comment:
This might be the wrong time to introduce this, in other words, ignore this... but:

```suggestion
  RATE = {
    NEGATIVE: -3.213,
    SMALL_POSITIVE: 0.5,
    MEDIUM_POSITIVE: 1.621,
    LARGE_POSITIVE: 2.475
  }
```

Reply:
Constants are already introduced in the [basics exercise](https://github.com/exercism/v3/blob/master/languages/ruby/exercises/concept/basics/.meta/example.rb) . Even though this exercise does not list the basics exercise as a prerequisite, it does list the numbers exercise as one. And since the numbers exercise depends on the basics exercise in the csharp track I assume the same will be the case for the Ruby track. Can't be sure though

Reply:
The introduction isn't constants here, but using a Hash.  Not sure it fits here.

Reply:
Yeah. Let's not introduce Hashes here.